### PR TITLE
#usCaESGS-App-Internationalization

### PR DESCRIPTION
**Pull request by Nkbtemmy**
**What does this PR do?**
    -This PR is about translating welcome message into 3 languages
**Description of Task to be completed?**
    -this PR enable user to select language of his/her choice (english, french and kinyarwanda) and see contents in his/her choice
**How should this be manually tested?**
   1. Clone this Repo if you don't have
   2. type git checkout `origin/ft_backend_internationalise_qSwfwWJp`
  3. install package( `npm i`) and then start server by writing `npm run dev`
  4. Open your postman and type in `GET:localhost:5000` by Default messages will displayed in english
     . then go to Header type in key field **Accept-Language** and type **2 first characters of language you choose** (ex:ki)
     . then click on Send to see what happen
 5.Also you can open your browser and type in URL **localhost:5000/?lang=fr**
     . you can change fr to ki or en

**Any background context you want to provide?**
    * en: English
    * fr: French
    * ki: kinyarwanda

**What are the relevant pivotal tracker stories?**
    [usCaESGS](https://trello.com/c/usCaESGS/20-support-multiple-languages)

**Screenshots (if appropriate)**
   Screen Shot 2020-09-28 at 15 47 50

### CODE REVIEW

**Review by karigirwa — DISMISSED** [review]

**Review by ncutixavier — DISMISSED** [review]

**Review by Niyiragira — DISMISSED** [review]

**Review by blaise82 — CHANGES_REQUESTED** [review]
update your folder structure
remove the language files from the utils folder 
create a new folder in the src fold called language under it add a confgi.js file for your configuration and a locales folder

File: package.json [hunk 1]
```diff
@@ -14,7 +14,9 @@
   "license": "ISC",
   "dependencies": {
     "body-parser": "^1.19.0",
+    "cookie-parser": "^1.4.5",
```
Comment (blaise82) [inline package.json:17]: why did you install cookie-parser

File: src/utils/international.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  extension: '.json',
+  queryParameter: 'lang', 
+  directoryPermissions: '755',
+  autoReload: true,
+  cookie: 'yourcookiename',
+  directory: __dirname + '/locales'
+});
+app.use(cookieParser());
+app.use(i18n.init);
+module.exports =app;
```
Comment (blaise82) [inline src/utils/international.js]: add a new line and use es6

**Review by blaise82 — CHANGES_REQUESTED** [review]
add a new line at the end of every line

File: package.json (hunk 1 above)
Comment (Nkbtemmy) [reply package.json:17]: I installed it , because i want to be able to use language selected to all other routers

**Comment by Niyiragira** [comment]
You have not rebased man. Your branch does not contain swagger.json, .env.example and others as in develop. So make sure you rebase.

**Comment by Niyiragira** [comment]
don't know if the case is for only me but manually testing I only get English

**Review by ncutixavier — APPROVED** [review]

**Review by Niyiragira — APPROVED** [review]

**Review by karigirwa — APPROVED** [review]

**Review by blaise82 — APPROVED** [review]

**Pull request merged**